src/js/08-theme.js

```diff
@@ -1,31 +1,20 @@
 /* global localStorage */
 ;(function () {
   'use strict'
-  const systemInitiatedDark = window.matchMedia('(prefers-color-scheme: dark)')
-  if (systemInitiatedDark.matches) {
-    document.documentElement.setAttribute('data-theme', 'dark')
+  let theme = localStorage.getItem('theme')
+  if (theme !== null) {
+    document.documentElement.setAttribute('data-theme', theme)
+  } else {
+    const systemDark = window.matchMedia('(prefers-color-scheme: dark)').matches
+    document.documentElement.setAttribute('data-theme',
+      systemDark ? 'dark' : 'light')
   }
```

Comment:
I think there's a bug here. Seems like if I have no theme set in localstorage, then click the button (to go from dark to light) nothing happens, and I see dark set in localstorage.
Here's one suggestion
```suggestion
  if (theme === null) {
    const systemDark = window.matchMedia('(prefers-color-scheme: dark)').matches
    theme = systemDark ? 'dark' : 'light'
  }
  document.documentElement.setAttribute('data-theme', theme)
```

Reply:
yeah you're right b/c i forgot to set the theme again in the else block. oops 😢 when you approve, can you merge and also rebuild the docker image? i'll handle the updates to the maintainer guides in both pt and dovah.